insta/src/App.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data'
+import SearchBar from "./components/SearchBar/searchBar";
+import PostContainer from './components/PostContainer/PostContainer'
+
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <SearchBar />
+        {dummyData.map(item =>{
```

Comment:
Instead of mapping the dummyData through the file directly try to set it to state and then map it from state